gym/envs/registration.py

```diff
@@ -441,14 +447,16 @@ def register(self, id: str, **kwargs) -> None:
             logger.warn(f"Overriding environment {id}")
         self.env_specs[id] = EnvSpec(id, **kwargs)
 
-    def _versions(self, namespace: str, name: str) -> Set[Tuple[int, str]]:
+    def _versions(self, namespace: str, name: str) -> Set[Tuple[Optional[int], str]]:
         # Get the set of versions under the requested namespace
         versions = set(
             map(
                 lambda version: (
                     int(version),
                     namespace,
-                ),
+                )
```

Comment:
Here we can do,
```py
lambda version: (
    version and int(version),
    namespace
)
```
and get rid of the conditional in the map. Note: `x and f(x)` will evaluate to None if `x is None`, otherwise it will evaluate `f(x)`.

Reply:
Smart!